app/controllers/OtherReasonController.scala

```diff
@@ -210,7 +210,10 @@ class OtherReasonController @Inject()(whenDidBecomeUnablePage: WhenDidBecomeUnab
       val journeyId = request.session.get(SessionKeys.journeyId).get
       upscanService.getAmountOfFilesUploadedForJourney(journeyId).map(
         filesUploaded => {
-          Ok(youHaveUploadedFilesPage(formProvider, radioOptionsToRender, postAction, filesUploaded))
+          if (filesUploaded < 5)
+            Ok(youHaveUploadedFilesPage(formProvider, radioOptionsToRender, postAction, filesUploaded))
+          else
+            Ok(youHaveUploadedFilesPage(formProvider, Seq.empty, postAction, filesUploaded))
```

Comment:
If you make the change I suggested in the view, you probably wouldn't need this

Reply:
(I can't submit a comment on the **onSubmitForUploadComplete**) but I suspect it will return a Bad Request when the form is not visible. It should be adapted to check that if the amount of files uploaded < 5 then do the form validation, else we should redirect the user to the next page